### DISCUSSION TITLE
How to delete metic series by label via an API?

### DISCUSSION THREAD

**Discussion by ntimo** [grafana/mimir#1712] [Help and support] [answered]
Hello,
I am currently searching for a way preferably an api to delete time series from Mimir. I thought `/api/v1/admin/tsdb/delete_series` might work but as it turns out it does not :D (404). I sadly also wasn't able to find some kind of docs for this. I need to delete some old series that have been scraped, in order to not mess up my dashboards.

Thanks for your help

**Comment by wilfriedroset** [comment, marked as answer]
Hello, I reckon this is not supported at the moment.
This issue seems to capture your need https://github.com/grafana/mimir/issues/88
